sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        g = sta / g_std
</pre><p>This is confusing, since it really should be the filter k, rather than g, which is the filtered stimulus. I think correct though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>